examples/Actor/DemoActor/Dockerfile

```diff
@@ -0,0 +1,28 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:6.0-jammy-chiseled-extra AS base
```

Comment:
It'd be nice to avoid having to maintain a `Dockerfile` going forward (or taking on VS dependencies just for the purpose of creating containers).  I'd suggest using the [built-in .NET CLI support for publishing applications as containers](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container?pivots=dotnet-8-0#publish-net-app), which should work even for our .NET 6 based example/sample projects.

Reply:
I removed the Dockerfile and used the integrated container support, i also updated the readme to reflect the changes